package.json

```
@@ -6,18 +6,22 @@
     "start": "docusaurus start",
     "build": "docusaurus build",
     "swizzle": "docusaurus swizzle",
-    "deploy": "docusaurus deploy"
+    "deploy": "docusaurus deploy",
+    "postinstall": "patch-package"
   },
   "dependencies": {
     "@docusaurus/core": "^2.0.0-alpha.48",
     "@docusaurus/preset-classic": "^2.0.0-alpha.48",
     "@docusaurus/theme-live-codeblock": "^2.0.0-alpha.39",
     "classnames": "^2.2.6",
+    "core-js": "3",
     "exoflex": "^3.3.0",
+    "patch-package": "^6.2.1",
+    "postinstall-postinstall": "^2.1.0",
     "react": "^16.8.4",
     "react-dom": "^16.8.4",
-    "react-native": "0.60.6",
-    "react-native-web": "^0.12.2"
+    "react-native-svg": "^12.0.3",
+    "react-native-web": "^0.11.0"
```

Shall we `0.11.7` instead?
```suggestion
    "react-native-web": "^0.11.7"
```

---

It's the same since we use ^ right?

---

Actually, I think it's better if we remove the caret and use specific version instead.